docs/cookbook.rst

```diff
@@ -23,14 +23,21 @@ The following example shows how to add the mixin to ``ContactView``::
 
 
     class MyContactView(FormMessagesMixin, ContactView):
-        form_invalid_message = _(u"There was en error in the contact form.")
         form_valid_message = _(u"Thank you for your message.")
+        form_invalid_message = _(u"There was en error in the contact form.")
```

Comment:
What's the change here? Now I see my typo (en -> an), but that's another matter.